app/src/main/java/com/example/danielmoreno/phonesettings/settings/SettingsListModelImplementation.java

```diff
@@ -0,0 +1,58 @@
+package com.example.danielmoreno.phonesettings.settings;
+
+import com.example.danielmoreno.phonesettings.R;
+import com.example.danielmoreno.phonesettings.settings.interfaces.SettingsListModels;
+import com.example.danielmoreno.phonesettings.settings.interfaces.SettingsListPresenter;
+import com.example.danielmoreno.phonesettings.settings.models.SettingModel;
+
+import java.util.ArrayList;
+
+public class SettingsListModelImplementation implements SettingsListModels {
+
+    private static String[] SETTINGS_NAMES_ARRAY = {
+            "Internet & Devices",
+            "Screen Brightness",
+            "Volume & Ringtone",
+            "Home Screen",
+            "Customize Phone",
+            "Android Settings"
+    };
+
+    private static int[] SETTINGS_IMAGES_RESOURCE_ARRAY = {
+            R.drawable.ic_volume,
+            R.drawable.ic_brightness,
+            R.drawable.ic_volume,
+            R.drawable.ic_home,
+            R.drawable.ic_mode_edit,
+            R.drawable.ic_android
+    };
+
+    private SettingsListPresenter mSettingsListPresenter;
+
+    public SettingsListModelImplementation() {
+
+    }
+
+    /**
+     * Sets the presenter for MVP
+     *
+     * @param mSettingsListPresenter Settings Presenter
+     */
+    public void setSettingsListPresenter(SettingsListPresenter mSettingsListPresenter) {
```

Comment:
Change the parameter name to settingsListPresenter instance of mSettingsListPresenter